src/components/DropdownButton.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { Button } from "library-simplified-reusable-components";
+import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
+
+export interface DropdownButtonProps {
+  mainContent: string | JSX.Element;
+  menuContent: string[] | JSX.Element[];
+  callback: (event: React.MouseEvent) => void | any;
```

Comment:
Why return `any`?

Reply:
Trying to build in flexibility for if/when we want to reuse the component elsewhere...